angularjs-1.5-basics/src/app/landing-page/_landing-page.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->